packages/contracts/deploy/src/utils/network.ts

```diff
@@ -1,15 +1,14 @@
-import promisify = require('es6-promisify');
+import {promisify} from '@0xproject/utils';
+import {Web3Wrapper} from '@0xproject/web3-wrapper';
+import * as _ from 'lodash';
 import * as Web3 from 'web3';
 
-import {Web3Wrapper} from './web3_wrapper';
-
 export const network = {
     async getNetworkIdIfExistsAsync(port: number): Promise<number> {
         const url = `http://localhost:${port}`;
         const web3Provider = new Web3.providers.HttpProvider(url);
-        const defaults = {};
-        const web3Wrapper = new Web3Wrapper(web3Provider, defaults);
-        const networkIdIfExists = await web3Wrapper.getNetworkIdIfExistsAsync();
-        return networkIdIfExists;
+        const web3Wrapper = new Web3Wrapper(web3Provider);
+        const networkId = web3Wrapper.getNetworkIdAsync();
+        return networkId;
```

Comment:
Let's get rid of this file completely. It's only called in two places, so let's refactor those to call `web3Wrapper` directly.